ament_index_python/package.xml

```diff
@@ -9,6 +9,12 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
```

Comment:
This should be a `buildtool` dependency since it is needed on the host platform (not the target platform).